.github/workflows/ci.yml

```diff
@@ -115,13 +115,14 @@ jobs:
           bazel test //tensorboard/summary/writer:event_file_writer_s3_test &&
           bazel test //tensorboard/compat/tensorflow_stub:gfile_fsspec_test &&
           bazel test //tensorboard/summary/writer:event_file_writer_fsspec_test
-      - name: 'Bazel: build Pip package (with TensorFlow only)'
-        if: matrix.tf_version_id == 'tf'
+      - name: 'Bazel: build Pip package (master branch TF build only)'
+        # Only bother creating the pip package if it will be updated in the next step.
+        if: matrix.tf_version_id == 'tf' && github.repository == 'tensorflow/tensorboard' && github.ref == 'refs/heads/master'
```

Comment:
Oh, I was confused about the github repository part until I saw the comment on line 126, below. Should we add the same clarification here?

Reply:
Ok, I've expanded the comment to explicitly direct the reader to the other comment. I didn't really want to replicate the explanation itself, since the point is really just to match the upload step.

Reply:
Oh... for some reason I thought "next step" was not referring to what this file is describing... but something after CI passes... which possibly doesn't make much sense... but I guess the new version of the comment is fine as well. Thanks for updating it.